### PR TITLE
🖼️ Add cover image to content layout

### PR DESCRIPTION
**Pull request by MeganSteenkamp**
Add cover image with a path pulled from markdown.

Test plan: Expect to see cover image as per screenshot.

Screenshot 2023-01-11 at 4 49 42 PM
Screenshot 2023-01-11 at 4 49 51 PM

### CODE REVIEW

File: content/guides/farmlands-pos-integration.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Farmlands POS Integration Guide
+img: /farmlands-pos-integration-cover.jpg
```
Comment (tom-blake) [inline content/guides/farmlands-pos-integration.md:3]: What was your reasoning for using a JPG over a PNG? I see the other image you used was a PNG

Comment (tom-blake) [inline content/guides/farmlands-pos-integration.md:3]: @MeganSteenkamp I think it's good that you used the front-matter to declare the image path, @AndyClifford will need this path to display this guide as a small card on the partner page 👍 

Is there a reason why it doesn't need to be the full path (include `/public`)?

Comment (AndyClifford) [reply content/guides/farmlands-pos-integration.md:3]: Is it possible to use the assets directory here? `~/assets/farmlands/pos-integration-cover.jpg`

Comment (MeganSteenkamp) [reply content/guides/farmlands-pos-integration.md:3]: @tom-blake I went to Unsplash and got the original image as the one from Figma wasn't large enough. Unsplash didn't have an option for the file type and downloaded it as a JPG.

Comment (MeganSteenkamp) [reply content/guides/farmlands-pos-integration.md:3]: @tom-blake @AndyClifford I have tried the assets directory but it didn't work 🤔. By default files in the `/public` directory are served at the root which is why I didn't need to specify the folder.

I see Nuxt has an image module if we did want to optimize images: https://v1.image.nuxtjs.org/get-started/

Comment (tom-blake) [reply content/guides/farmlands-pos-integration.md:3]: @MeganSteenkamp Yeah it's not working for me either. [Found this](https://github.com/nuxt/content/issues/106#issuecomment-1215530462) comment that suggests copying the files to `public` or using the module you mentioned

Comment (tom-blake) [reply content/guides/farmlands-pos-integration.md:3]: I see - thanks! Maybe let the designers know if Unsplash isn't what they'd do :)

**Review by tom-blake — APPROVED** [review]

**Pull request merged**